fastjet/internal/delaunay/delaunay.go

```diff
@@ -68,25 +70,29 @@ func (d *Delaunay) Triangles() []*Triangle {
 // Insert inserts the point into the triangulation. It returns the points
 // whose nearest neighbor changed due to the insertion. The slice may contain
 // duplicates.
-func (d *Delaunay) Insert(p *Point) (updatedNearestNeighbor []*Point) {
+func (d *Delaunay) Insert(p *Point) (updatedNearestNeighbor points) {
```

Comment:
having an exported method returning a value of type `points` which isn't itself exported is a bit cumbersome.

either export `points` or re-convert the `points` to `[]*Point` when returning from this method.
Looking at the `points` type, it only has one method, that is not exported.
Thus, I would go with the latter (keep `[]*Point` in the method signature and work with `points` inside the method, re-converting to `[]*Point` when returning.)

Reply:
done